CHANGELOG.md

```diff
@@ -151,7 +151,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Support for input forms ([#790](https://github.com/livebook-dev/livebook/pull/790))
 - Completion for struct keys ([#793](https://github.com/livebook-dev/livebook/pull/793))
 - Support for custom JavaScript widgets ([#826](https://github.com/livebook-dev/livebook/pull/826))
-- Introduced a dedicated Explore subsection for Kino guides ([#830](https://github.com/livebook-dev/livebook/pull/830))
+- Introduced a dedicated Learn subsection for Kino guides ([#830](https://github.com/livebook-dev/livebook/pull/830))
```

Comment:
Please revert the changes to `CHANGELOG.md`, it is for keeping track of changes as they happened in the past, so we only add new entries as we make new releases :)